pkgs/development/libraries/rinutils/default.nix

```diff
@@ -7,22 +7,15 @@
 
 stdenv.mkDerivation rec {
   pname = "rinutils";
-  version = "0.10.2";
+  version = "0.10.3";
 
   src = fetchurl {
```

Comment:
Can we use `fetchFromGitHub`?

Reply:
The upstream-packaged release tarball we’re getting here also contains a CMake module from https://github.com/shlomif/shlomif-cmake-modules that’s not in the Git tree. Figuring out which version of that is needed would complicate version bumps somewhat, but nothing too onerous. I would appreciate any help or references for writing unpackPhase for a package with two entries in `srcs`, though, because I’ve never succeeded at that.